content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -184,7 +184,7 @@ it does the following when a Pod is created:
      `/var/run/secrets/kubernetes.io/serviceaccount`.
      For Linux containers, that volume is mounted at `/var/run/secrets/kubernetes.io/serviceaccount`;
      on Windows nodes, the mount is at the equivalent path.
-1. If the spec of the incoming Pod does already contain any `imagePullSecrets`, then the
+1. If the spec of the incoming Pod doesn't already contain any `imagePullSecrets`, then the
```

Comment:
From the code implementation, your modification is correct.
```
4. If the pod does not contain any ImagePullSecrets, the ImagePullSecrets of the service account are added.

	if len(pod.Spec.ImagePullSecrets) == 0 {
		pod.Spec.ImagePullSecrets = make([]api.LocalObjectReference, len(serviceAccount.ImagePullSecrets))
		for i := 0; i < len(serviceAccount.ImagePullSecrets); i++ {
			pod.Spec.ImagePullSecrets[i].Name = serviceAccount.ImagePullSecrets[i].Name
		}
	}

```
https://github.com/kubernetes/kubernetes/blob/55181b72a2c0aacb344998460b08c708cd069bf1/plugin/pkg/admission/serviceaccount/admission.go#L98